src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -201,6 +203,8 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::pass::MVN6Decomposition>();
         manager.register_pass<ov::pass::BroadcastTransition>();
 
+        manager.register_pass<ov::pass::ConstantFolding>();
+        manager.register_pass<ov::pass::RemoveUselessConvertLike>();
```

Comment:
Why it can't be called after convert precision pass (in common optimizations for example)?

Reply:
for some reason i was getting cumbersome graph with a lot of `ConvertLike`s and they were blocking mixed precision marking. ![image](https://github.com/openvinotoolkit/openvino/assets/5703039/5e0a00ac-6279-4a09-ada0-0654e186ce6e)

Such graph was visible only on GPU, CPU called ConstantFold multiple time and in their `transformation_pipeline.cpp` graph looked good. That was the reason why i added it only for GPU. But now i completely removed this c++ code. If we call the new utility `openvino.tools.ovc.convert_movel` then graph is normal without redundant ConvertLikes. 

So now only python code is in this PR.